index.html

```diff
@@ -129,10 +129,12 @@ <h1 id="title">Decentralized Identifier Working Group Charter</h1>
 
         <p>
 URL identifiers in use on the Web today (2019) require that the identifier is
-leased from an authority such as a Domain Name Registrar. The advent of
-Blockchains and Decentralized Ledger Technologies have led to innovations
-around a new type of URL, called a Decentralized Identifier (DID). DIDs have
-benefits over more traditional URLs:
+leased from an authority such as a Domain Name Registrar. An identifier that
+does not need to be leased from an authority, but instead may be created and
+used without a central authority would be a Decentralized Identifier (DID).
```

Comment:
Wording is a bit strange. 

Alternate: "A Decentralized Identifier (DID) is an identifier that does not need to be leased; its creation and usage is possible without a central authority to manage it."

That said, I'm fine w/ it either way.

Reply:
addressed in [aee4288](https://github.com/w3c-ccg/did-wg-charter/pull/27/commits/aee4288b3302a701b9fa1b619473f59c5fe1a1c0)